### DISCUSSION TITLE
账号鉴权失败

### DISCUSSION THREAD

**Discussion by SevenUp2719** [Richasy/Bili.Uwp#1633] [问答]
![image](https://user-images.githubusercontent.com/123624466/214832518-12efb4f5-dad1-4ea9-b5c6-364e59698860.png)

**Comment by xinghuo0922** [comment]
我也是，已经登不上了😭

**Comment by kitzhang2019** [comment]
一样的，我也是鉴权失败

**Comment by Brokencrystal** [comment]
一样是这个问题

**Comment by Aurora-Simba** [comment]
此项目已停更

**Comment by aizhimou** [comment]
我也登陆不上了，这个项目是不更新了吗？ @Richasy
